src/django/api/facilities_download_view_set.py

```diff
@@ -84,3 +111,27 @@ def __can_user_download_over_limit(
             return False
 
         return True
+
+    @staticmethod
+    def reset_facility_download_limits_next_month(
+        facility_download_limit,
+    ):
+        facility_download_limit.last_download_time = timezone.now()
+        facility_download_limit.download_count = 0
+        facility_download_limit.save()
+
+    @staticmethod
+    def is_same_month_as_last_download(last_download_time) -> bool:
```

Comment:
I don't think we really need this method, we could handle this logic in the view function.